### ISSUE TITLE
Recurrings do not create related jobs for queue

### ISSUE THREAD

**Issue by oliverjkb** [range-of-motion/budget#20]
#### Actual behavior

When a `recurring` transaction is entered, the model will be persisted to the database as intended.  
However, no corresponding job will be despatched to the queue, hence recurrings will _never_ be reviewed, breaking the intended use case.  
Read: Recurrings will _never_ find their way into the transactions. 

#### Expected behavior 

When entering a recurring transaction, two actions should take place:  
* save an instance of `recurring` database
* save an instance of `job` to database

#### Steps to reproduce 

* Select `Create Transaction` 
* Choose `Spending` 
* Tick `This is a recurring […]` 
* Save

#### Extended Issue 

Unfortunately, adding a `ProcessRecurrings::dispatch($recurring);` will only in part resolve the issue.  
Since a `ProcessRecurrings`-job will only fire once and will then be removed, there will be next to no Recurrings ever going to find their way to the transactions. 

An additional scheduled Task should be created to add a corresponding Job to the queue for every active Recurring in the database _every single day_.

**Comment by oliverjkb** [comment]
So what I did not have in mind was, that there were scheduled job creations in place. 
They were just not mentioned in the README, so I never started the scheduler ^^. 

Non the less, I updated the schedule for this particular job to daily, since instant job creation will handle the creation of a spending with the job that is created now, when entering the recurring.

Also having a setup with tens of hundres of users, creating hundreds of recurrings would likely not finisch within a minute, hence creating race conditions when creating new jobs every minute. 

README.md got an update so new users can setup the cronjob required to use recurrings and weekly reports.
